docs/configuration/configuration.md

```diff
@@ -1342,6 +1342,7 @@ Available meta labels:
 * `__meta_kubernetes_ingress_labelpresent_<labelname>`: `true` for each label from the ingress object.
 * `__meta_kubernetes_ingress_annotation_<annotationname>`: Each annotation from the ingress object.
 * `__meta_kubernetes_ingress_annotationpresent_<annotationname>`: `true` for each annotation from the ingress object.
+* `__meta_kubernetes_ingress_class`: Class from the ingress spec, if present. Defaults to `""`.
```

Comment:
No need to elaborate on the default I think, and `""` is equivalent to the non-existence, so I think it would be more confusing than helpful to call this out explicitly.